client/question/services/FeedbackGeneratorService.js

```diff
@@ -62,6 +93,40 @@ tie.factory('FeedbackGeneratorService', [
     var previousRuntimeErrorString = '';
 
     /**
+     * Pseudo-randomly returns an int between min and max (inclusive).
+     *
+     * @param {number} min Minimum number for range.
+     * @param {number} max Maximum number for range.
+     * @returns {number}
+     * @private
+     */
+    var _getRandomInt = function(min, max) {
+      min = Math.ceil(min);
+      max = Math.floor(max);
+      return Math.floor(Math.random() * (max - min)) + min
+    };
+
+    /**
+     * Randomly selects feedback string from an object containing variations
+     * of a given feedback type.
+     *
+     * @param {Object.<string, Array.<string>>} correctnessFeedbackText
+     * @param {string} type Correctness feedback type.
+     * @returns {string}
+     * @private
+     */
+    var _getCorrectnessFeedbackString = function(correctnessFeedbackText,
```

Comment:
Nit: Insert line break after '('.

Also, I still don't understand why correctnessFeedbackText is being passed in. Can't this method refer directly to CORRECTNESS_FEEDBACK_TEXT?

Reply:
Changed line break.

Re: correctnessFeedbackText vs CORRECTNESS_FEEDBACK_TEXT, I just forgot to change the variable name to the new one. Fixed.